### PR TITLE
Heinz: slides

### PR DESCRIPTION
**Pull request by shiltemann**
Hi @CicoZhang :) I started the import of your slides here, please feel free to comment/modify

### CODE REVIEW

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+- The number of genes in all the microbes in one person's microbiome is ~10-200 times the number of genes in the human genome.
+- The microbiome may weigh as much as 200 grams
+
+
+.image-50[ ![](../../../metagenomics/images/human_microbiome.png)]
+
+.footnote[[Sender et al 2016, PLoS ONE](https://doi.org/10.1371/journal.pbio.1002533)]
+
+???
+The most common objective is to ide
```
Comment (erasche) [inline topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: What does "ide" mean here?

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+]
+
+---
+
+## Some bacterial species are difficult to culture
+
+![petri dish with bacterial cultures](../../images/heinz/petridish.png)
+
+- Such bacteria may not be found in current reference databases
+- The only way to get hold of each species may be through direct sequencing of environmental samples
```
Comment (erasche) [inline topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: s/get hold of each/obtain/

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+
+Subsequently, the Heinz algorithm can find the subnetwork that is most deregulated.
+
+---
+
+# Differential Expression Analysis
+
+![differential expression analysis plot](../../images/heinz/dea.png)
+
+Each dot's (function's) change in expression has a P-value associated with it
```
Comment (erasche) [inline topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: the ending `it` runs off the slide.

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html [hunk 4]
```diff
@@ -0,0 +1,528 @@
...
+First, we need to convert the p-values to weight. Here we think about a p-value distribution on a set that has no signal. In this case we expect a uniform distribution of p-values between 0 and 1. Note that if you generate many p-values for a data set, it is always helpful to check the distribution to see if there is any signal in the data.
+
+
+---
+
+# P-value distribution: signal vs noise
+
+![Beta uniform mixture (BUM) model](../../images/heinz/bum.png "P-value histogram of 7105 KO fold changes")
+
+<span style="color:blue">Noise (blue)</span> and <span style="color:red">signal (red)</span> in an expected P-value distribution
```
Comment (erasche) [inline topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: After `beta-` off screen

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html [hunk 5]
```diff
@@ -0,0 +1,528 @@
...
+---
+
+# P-value to weight transformation
+
+![](../../images/heinz/pvalue-to-weight.png)
+
+$$ FDR = \frac{C}{A+C}$$
+
+- **A:** True positives, **C:** False positives
+- False discovery rate (**FDR**) -> a threshold P-value
```
Comment (erasche) [inline topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: Math pushes this one offscreen as well

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html [hunk 6]
```diff
@@ -0,0 +1,528 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "Network Analysis with Heinz"
+questions:
+  - "What is metatranscriptomics"
+  - "Which pathways are potentially contributing to dental caries?"
+objectives:
+  - "Know some background knowledge about microbiome"
```
[the remark this answers is not in the document]
Comment (CicoZhang) [reply topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: Done.

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html (hunk 1 above)
Comment (CicoZhang) [reply topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: Fixed. Thx.

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html (hunk 2 above)
Comment (CicoZhang) [reply topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: Fixed. Thx.

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html (hunk 3 above)
Comment (CicoZhang) [reply topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: Rephrased it. Thx.

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html (hunk 5 above)
Comment (CicoZhang) [reply topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: Like this on my screen?
image

File: topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html (hunk 4 above)
Comment (CicoZhang) [reply topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: ?
image

Comment (erasche) [reply topics/transcriptomics/tutorials/network-analysis-with-heinz/slides.html]: Ok, looks like I had a font issue, I will remove these issues.

**Pull request merged**